### PR TITLE
Proposing a Hacktoberfest Issue template

### PR DESCRIPTION
**Pull request by TheBusyBiscuit**
We normally only use the Issues Tracker for bug reports. But as Hacktoberfest approaches, I think we should create some labelled issues which feature implementation gaps, some approved suggestions, bugs or refactoring plans that we wanna share with the Open-Source community.
These issues should only be opened by us, organization members, if someone wants to give another idea, they should go the usual suggestions-route on discord.

I want these issues to be as helpful and beginner-friendly as possible, here is my proposed solution.<br>
Feel free to give suggestions on what to add etc...

<hr>

This Issue is part of [Hacktoberfest](https://hacktoberfest.digitalocean.com/) - A yearly event which encourages participation in the open-source community. [Sign up on their website](https://hacktoberfest.digitalocean.com/) and submit **four pull requests** to any public open-source project on GitHub and earn a limited-edition T-shirt!

## :mag_right: Scope
The following bullet points explain what the scope of this feature should be.
It should give a general idea of what we want, you can of course deviate from this as needed.

* Do something
* Do another thing
* Oh also this one

## :anchor: Difficulty
Here is our honest estimate on how difficult (on a scale of 1-5) the implementation may be:

:large_blue_circle::white_circle::white_circle::white_circle::white_circle:

## :construction: Technical Challenges
These are some challenges which may need to be overcome, we wanna be as transparent as possible, so here is some guidance on what may present itself as an obstacle.

* Remove Herobrine
* No rickrolls

## :memo: Relevant Classes or Snippets
Here are some classes or code snippets which we think might help you get a better understanding where to look for in this gigantic codebase

* lorem ipsum

## :book: Useful Resources
If you need help on how to get started, maybe try looking into the following resources!

* Hacktoberfest
    * [Getting started with Hacktoberfest](https://hacktoberfest.digitalocean.com/details#get-started)
    * [Hacktoberfest FAQ](https://hacktoberfest.digitalocean.com/faq)
* GitHub/Open-Source
    * [How to contribute to Open-Source](https://opensource.guide/how-to-contribute/)
    * [Working with forks](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/working-with-forks)
    * [Creating a Pull Request](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request-from-a-fork)
* Slimefun
    * [Contributing to Slimefun](https://github.com/Slimefun/Slimefun4/blob/master/.github/CONTRIBUTING.md)
    * [Code of Conduct](https://github.com/Slimefun/Slimefun4/blob/master/.github/CODE_OF_CONDUCT.md)

<hr>

If you want to work on this, simply comment down below and state your interests! Please also comment again if you have changed your mind. Anyone is allowed to discuss in the comments below, feel free to collaborate and work together :heart: <br>
You can always ask for help here if you get stuck.

### CODE REVIEW

**Review by Sfiguz7 — DISMISSED** [review]

**Comment by Sfiguz7** [comment]
Prep-tember at its finest

**Comment by TheBusyBiscuit** [comment]
Yep, written a Code of Conduct, rewrote the CONTRIBUTING.md file, now setting up issue templates 🎉

**Review by LinoxGH — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/hacktoberfest-issue.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  an issue for you if applicable."
+title: ''
+labels: Hacktoberfest
+assignees: ''
+
+---
+
+<!-- READ ME FIRST -->
+<!-- If you wanna submit an idea for Hacktoberfest, please go to our discord server -->
+<!-- And post your ideas there, this issue format is only for internal purposes only. -->
```
Comment (poma123) [inline .github/ISSUE_TEMPLATE/hacktoberfest-issue.md:13]: Only two "only"s (only) :(

**Review by poma123 — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/hacktoberfest-issue.md (hunk 1 above)
Comment (TheBusyBiscuit) [reply .github/ISSUE_TEMPLATE/hacktoberfest-issue.md:13]: Oh XD thanks

**Review by poma123 — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/hacktoberfest-issue.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+assignees: ''
+
+---
+
+<!-- READ ME FIRST -->
+<!-- If you wanna submit an idea for Hacktoberfest, please go to our discord server -->
+<!-- And post your ideas there, this issue format is for internal purposes only. -->
+<!-- If we like your idea, we will happily convert it into an actual issue. -->
+
+This Issue is part of [Hacktoberfest](https://hacktoberfest.digitalocean.com/) - A yearly event which encourages participation in the open-source community. [Sign up on their website](https://hacktoberfest.digitalocean.com/) and submit **four pull requests** to any public open-source project on GitHub and earn a limited-edition T-shirt!
```
Comment (WalshyDev) [inline .github/ISSUE_TEMPLATE/hacktoberfest-issue.md:16]: or plant a tree!

**Review by poma123 — APPROVED** [review]
Approved again, Cookie!

**Review by WalshyDev — COMMENTED** [review]
LGTM

**Review by poma123 — APPROVED** [review]

**Pull request merged**